libcudacxx/include/cuda/std/detail/libcxx/include/bitset

```diff
@@ -129,932 +137,889 @@ template <size_t N> struct hash<std::bitset<N>>;
-    unsigned long to_ulong() const;
-    _LIBCUDACXX_INLINE_VISIBILITY
-    unsigned long long to_ullong() const;
-    template <class _CharT, class _Traits, class _Allocator>
-        basic_string<_CharT, _Traits, _Allocator> to_string(_CharT __zero = _CharT('0'),
-                                                            _CharT __one = _CharT('1')) const;
-    template <class _CharT, class _Traits>
-        _LIBCUDACXX_INLINE_VISIBILITY
-        basic_string<_CharT, _Traits, allocator<_CharT> > to_string(_CharT __zero = _CharT('0'),
-                                                                    _CharT __one = _CharT('1')) const;
-    template <class _CharT>
-        _LIBCUDACXX_INLINE_VISIBILITY
-        basic_string<_CharT, char_traits<_CharT>, allocator<_CharT> > to_string(_CharT __zero = _CharT('0'),
-                                                                                _CharT __one = _CharT('1')) const;
-    _LIBCUDACXX_INLINE_VISIBILITY
-    basic_string<char, char_traits<char>, allocator<char> > to_string(char __zero = '0',
-                                                                      char __one = '1') const;
-    _LIBCUDACXX_INLINE_VISIBILITY
-    size_t count() const noexcept;
-    _LIBCUDACXX_INLINE_VISIBILITY constexpr size_t size() const noexcept {return _Size;}
-    _LIBCUDACXX_INLINE_VISIBILITY
-    bool operator==(const bitset& __rhs) const noexcept;
-    _LIBCUDACXX_INLINE_VISIBILITY
-    bool operator!=(const bitset& __rhs) const noexcept;
-    bool test(size_t __pos) const;
-    _LIBCUDACXX_INLINE_VISIBILITY
-    bool all() const noexcept;
-    _LIBCUDACXX_INLINE_VISIBILITY
-    bool any() const noexcept;
-    _LIBCUDACXX_INLINE_VISIBILITY bool none() const noexcept {return !any();}
-    _LIBCUDACXX_INLINE_VISIBILITY
-    bitset operator<<(size_t __pos) const noexcept;
-    _LIBCUDACXX_INLINE_VISIBILITY
-    bitset operator>>(size_t __pos) const noexcept;
-
-private:
-
-    _LIBCUDACXX_INLINE_VISIBILITY
-    size_t __hash_code() const noexcept {return base::__hash_code();}
-
-    friend struct hash<bitset>;
-};
-
-template <size_t _Size>
-template<class _CharT, class>
-bitset<_Size>::bitset(const _CharT* __str,
-                      typename basic_string<_CharT>::size_type __n,
-                      _CharT __zero, _CharT __one)
-{
+  typedef typename base::reference reference;
+  typedef typename base::const_reference const_reference;
+
+  // 23.3.5.1 constructors:
+  _LIBCUDACXX_HIDE_FROM_ABI _LIBCUDACXX_INLINE_VISIBILITY constexpr bitset() noexcept {}
+  _LIBCUDACXX_HIDE_FROM_ABI _LIBCUDACXX_INLINE_VISIBILITY constexpr bitset(unsigned long long __v) noexcept
+      : base(__v)
+  {}
+  template <class _CharT, class = __enable_if_t<_IsCharLikeType<_CharT>::value>>
+  _LIBCUDACXX_HIDE_FROM_ABI _LIBCUDACXX_INLINE_VISIBILITY _CCCL_CONSTEXPR_CXX14 explicit bitset(
+    const _CharT* __str,
+#if (_LIBCUDACXX_STD_VER < 2026 && defined(_LIBCUDACXX_HAS_STRING)) || defined(_LIBCUDACXX_HAS_STRING_VIEW)
```

Comment:
Important: I moved all of CCCL to `_CCCL_STD_VER` the old macro is just there for backwards compat

Reply:
Ack, I must've missed this one on the list of macros changing because I wasn't touching it and got no conflicts around it :sweat_smile: